src/components/Image/Image.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Image = (props) => {
+  const { src, classes } = props;
```

Comment:
classes? or className? Are you passing many classes or a single className?

Reply:
Yes, there is the possibility of passing more than one classes. And this is just a variable nothing else. We can still change it if it is confusing but that was our idea.